docs/custombuilders.md

```diff
@@ -3,37 +3,37 @@
 kpack provides the experimental CustomBuilder and CustomClusterBuilder resources to define and create [Cloud Native Buildpacks builders](https://buildpacks.io/docs/using-pack/working-with-builders/) all within the kpack api. 
 This allows more granular control of the buildpacks and buildpack versions utilized without relying on pre-existing Builder resources. 
 
-Before creating CustomBuilders you will need to create the Stack and Store resources described below. 
+Before creating CustomBuilders you will need to create the ClusterStack and ClusterStore resources described below. 
 
-### <a id='store'></a>Store
+### <a id='clusterstore'></a>ClusterStore
```

Comment:
We have a classic documentation problem here. If we merged this the "main" branch would contain "ClusterStacks" but, the documenation would be incompatible with the most recent released version.

Reply:
pushed docs to other branch